language/move-vm/types/src/values/values_impl.rs

```diff
@@ -3461,7 +3461,7 @@ pub mod prop {
             L::U32 => any::<u32>().prop_map(Value::u32).boxed(),
             L::U64 => any::<u64>().prop_map(Value::u64).boxed(),
             L::U128 => any::<u128>().prop_map(Value::u128).boxed(),
-            L::U256 => any::<u256>().prop_map(Value::u256).boxed(),
+            L::U256 => any::<u256::U256>().prop_map(Value::u256).boxed(),
```

Comment:
👀 
What is this change exactly?

Reply:
`u256` is a module so the code doesn't actually compile. `u256::U256` is the type we have in `move-core-types`.

This did not get detected by CI since the code is behind the "fuzzing" feature flag. <= This is a separate issue we'd better fix

Reply:
This is a lingering change from u256. This piece of code couldn't compile previously IIUC because it was under the fuzzing feature flag, which our CI right now don't check if it is compilable.